Sources/SPMTestSupport/MockRegistry.swift

```diff
@@ -94,13 +107,21 @@ public class MockRegistry {
     private func handleRequest(request: HTTPClient.Request) throws -> HTTPClient.Response {
         let routeComponents = request.url.absoluteString.dropFirst(Self.mockRegistryURL.absoluteString.count + 1).split(separator: "/")
         switch routeComponents.count {
+        case _ where routeComponents[0].hasPrefix("identifiers?url="):
+            guard let query = request.url.query else {
+                throw StringError("invalid url: \(request.url)")
+            }
+            guard let sourceControlURL = URL(string: String(query.dropFirst(4))) else {
```

Comment:
Are these four characters always the `url=` mentioned above, and therefore no additional check is needed for what they are?

Reply:
this is in the testing infrastructure so should be safe enough